src/language_server_protocol.rs

```diff
@@ -967,15 +971,32 @@ impl LanguageClient {
     }
 
     fn try_handle_command_by_client(&self, cmd: &Command) -> Result<bool> {
-        match cmd.command.as_str() {
-            java::command::APPLY_WORKSPACE_EDIT => self.handle_java_command(cmd),
-            rust_analyzer::command::RUN
-            | rust_analyzer::command::RUN_SINGLE
-            | rust_analyzer::command::SHOW_REFERENCES
-            | rust_analyzer::command::SELECT_APPLY_SOURCE_CHANGE
-            | rust_analyzer::command::APPLY_SOURCE_CHANGE => self.handle_rust_analyzer_command(cmd),
-
-            _ => Ok(false),
+        let filetype: String = self.vim()?.eval("&filetype")?;
+        let enabled_extensions = self.get(|state| state.enable_extensions.clone())?;
+        if !enabled_extensions
+            .unwrap_or_default()
+            .get(&filetype)
+            .cloned()
+            .unwrap_or_default()
```

Comment:
This doesn't seem consistent with doc.

If the config is not found, i.e., `null`, the config will be default to `{}`. Then entry is not found and default to `false`, causing this method return `Ok()` directly, whereas based on the doc, it should try language extension to handle.

I understand the return value of this method is confusing (true for unsuccessful handle and false for successful handle). Maybe can make it more explicit.

Reply:
Ah yes, you are correct, nice catch. Fixed now!

Yeah the return value is a little awkward. I'll try and come up with something more expressive after this one.